components/script/script_thread.rs

```diff
@@ -1526,6 +1535,33 @@ impl ScriptThread {
         document.run_the_animation_frame_callbacks();
     }
 
+    /// Handles firing of transition events.
+    #[allow(unsafe_code)]
+    fn handle_transition_event(&self, unsafe_node: UnsafeNode, name: String, duration: f64) {
+        let node = unsafe { ServoLayoutNode::from_unsafe(&unsafe_node) };
```

Comment:
Isn't this the script thread? Seeing ServoLayoutNode here feels weird anyway, I agree.